test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc

```diff
@@ -76,6 +88,37 @@ void UberFilterFuzzer::perFilterSetup(const std::string& filter_name) {
         .WillOnce(Invoke([&](const envoy::config::core::v3::GrpcService&, Stats::Scope&, bool) {
           return std::move(async_client_factory_);
         }));
+    read_filter_callbacks_->connection_.local_address_ = pipe_addr_;
+    read_filter_callbacks_->connection_.remote_address_ = pipe_addr_;
+  } else if (filter_name == NetworkFilterNames::get().HttpConnectionManager) {
```

Comment:
The HTTP connection manager has its own fuzzer, with a reasonably complex set of mocks to allow for high coverage:

  envoy/src/test/common/http/conn_manager_impl_fuzz_test.cc

Fuzzing it here is fine, but I would include a comment saying that there is a fuzzer just for this filter, and including a link or code path.

Reply:
> The HTTP connection manager has its own fuzzer, with a reasonably complex set of mocks to allow for high coverage:
> 
> envoy/src/test/common/http/conn_manager_impl_fuzz_test.cc
> 
> Fuzzing it here is fine, but I would include a comment saying that there is a fuzzer just for this filter, and including a link or code path.

That makes sense. Should I add it here or add it at the filter name list?

Reply:
The filter name list seems best to me.

Reply:
> The filter name list seems best to me.

Added! Thanks!